### DISCUSSION TITLE
Allow searching by actor, director, etc

### DISCUSSION THREAD

**Discussion by vladola** [sct/overseerr#1608] [Feature Requests]
As per title, it would be nice to be able to find anything with X actor, be it movie or tv show.

**Comment by sct** [comment]
Maybe I don't understand what you want here. You can already search by cast/crew names in the search?

**Reply by vladola** [reply]
Actually, you're right, it's already there. I must have had a typo in my search which returned none, hence I assumed it wasn't supported. My bad!
